package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "create-cloud-deploy-release",
+  "version": "0.0.1",
+  "description": "Github Action: Create Cloud Deploy Release",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "ncc build -m src/main.ts -o dist/main",
+    "lint": "eslint src/ tests/ --ext .ts,.tsx",
+    "format": "eslint src/ tests/ --ext .ts,.tsx --fix",
+    "test": "mocha -r ts-node/register -t 150s 'tests/unit/*.test.ts' --exit",
+    "e2e-tests": "mocha -r ts-node/register -t 150s 'tests/e2e.test.ts' --exit"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google-github-actions/create-cloud-deploy-release.git"
+  },
+  "keywords": [
+    "actions",
+    "google",
+    "cloud deploy",
+    "cloud",
+    "deploy",
+    "containers",
+    "delivery",
+    "pipeline"
+  ],
+  "author": "Google LLC",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/google-github-actions/create-cloud-deploy-release/issues"
+  },
+  "homepage": "https://github.com/google-github-actions/create-cloud-deploy-release#readme",
+  "dependencies": {
```

Comment:
Can you make sure all these are up to date - you can use `ncu` or a similar tool to automate.